packages/now-next/src/index.ts

```diff
@@ -486,6 +487,16 @@ export const build = async ({
         // User redirects
         ...redirects,
 
+        // Next.js pages, `static/` folder, reserved assets, and `public/`
+        // folder
+        { handle: 'filesystem' },
+
+        ...rewrites,
+        // Dynamic routes
+        // TODO: do we want to do this?: ...dynamicRoutes,
```

Comment:
Dynamic routes would need to be after handle: rewrite here. This todo doesn't seem to make that clear

Reply:
This TODO is from before and seems unlikely to be added, I can update it in case it is as a reminder